exonum-java-binding-core/src/main/java/com/exonum/binding/service/NodeFake.java

```diff
@@ -86,8 +88,9 @@ public NodeFake(MemoryDb database, byte[] publicKey) {
    * @throws NullPointerException if the transaction is null
    */
   @Override
-  public void submitTransaction(Transaction transaction) {
+  public HashCode submitTransaction(RawTransaction transaction) {
```

Comment:
👍 , needs to be improved. I wouldn't say that the documentation in the interface works here, because it doesn't actually submit a transaction with the given parameters anywhere, but just returns a zero hash. But that must be clarified in the docs.

Also, what is less confusing — returning a zero hash always (for different parameters), which might not work in all situations (e.g., you submit _several_ txs and store the returned hashes in a set/map); or returning a hash of `RawTransaction` — not exactly what such method shall return, but who relies on hashes of some internal message packet anyway?